.github/workflows/train.yml

```diff
@@ -6,6 +6,10 @@ on:
       - master
     paths:
       - 'code/*'
+  
+  repository_dispatch:
+    types: [run-status-changed_dummy,data-drift-detected]
```

Comment:
what is this event _dummy?

Reply:
right now run-status-changed event is being trigerred multple times on running train.yml and is causing several workflow triggers to train.yml  causing a loop which triggers more workflows.
When that is handled this can be changed to actual run-status-changed event coming from function app

Reply:
then you should not execute the workflow for that. you should only have 1 workflow which listen to it